redirects.yaml

```diff
@@ -315,3 +315,117 @@ redirects:
+  - source: /image-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/image-alt
+  - source: /input-image-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/input-image-alt
+  - source: /label/
+    destination: https://dequeuniversity.com/rules/axe/4.4/label
+  - source: /link-name/
+    destination: https://dequeuniversity.com/rules/axe/4.4/link-name
+  - source: /object-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/object-alt
+  - source: /color-contrast/
+    destination: https://dequeuniversity.com/rules/axe/4.4/color-contrast
+  - source: /definition-list/
+    destination: https://dequeuniversity.com/rules/axe/4.4/definition-list
+  - source: /dlitem/
+    destination: https://dequeuniversity.com/rules/axe/4.4/dlitem
+  - source: /list/
+    destination: https://dequeuniversity.com/rules/axe/4.4/list
+  - source: /listitem/
+    destination: https://dequeuniversity.com/rules/axe/4.4/listitem
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /layout-table/
+    destination:
+  - source: /td-headers-attr/
+    destination: https://dequeuniversity.com/rules/axe/4.4/td-headers-attr
+  - source: /th-has-data-cells/
+    destination: https://dequeuniversity.com/rules/axe/4.4/th-has-data-cells
+  - source: /meta-refresh/
+    destination: https://dequeuniversity.com/rules/axe/4.4/meta-refresh
+  - source: /meta-viewport/
+    destination: https://dequeuniversity.com/rules/axe/4.4/meta-viewport
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /audio-caption/
+    destination:
+  - source: /video-caption/
+    destination: https://dequeuniversity.com/rules/axe/4.4/video-caption
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /video-description/
+    destination:
+  - source: /html-has-lang/
+    destination: https://dequeuniversity.com/rules/axe/4.4/html-has-lang
+  - source: /html-lang-valid/
+    destination: https://dequeuniversity.com/rules/axe/4.4/html-lang-valid
+  - source: /valid-lang/
+    destination: https://dequeuniversity.com/rules/axe/4.4/valid-lang
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /logical-tab-order/
+    destination:
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /focusable-controls/
+    destination:
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /interactive-element-affordance/
+    destination:
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /managed-focus/
+    destination:
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /focus-traps/
+    destination:
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /custom-controls-labels/
+    destination:
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /custom-control-roles/
+    destination:
```

Comment:
The rest of these are so-called manual accessibility audits, things we ask the user to check because the automated tests can't. As a result there are no corresponding axe documents, so I'll need to migrate from web.dev.

Completely forgot about these :/